package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-router-proxy-loader",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

Comment:
Lets go to `0.4.0` for `react-router@1.x`.  That will give us space to have `0.3.1` for bug fixes on `react-router@0.x`